docs/src/content/docs/ucs/07.mdx

```diff
@@ -0,0 +1,248 @@
+
+## Motivation
+
+Token identification across blockchains presents several challenges:
+
+1. **Namespace Collisions**: Different chains may have tokens with identical contract addresses or denominations
+2. **Ecosystem Diversity**: Each blockchain ecosystem uses different token standards (ERC20, CW20, Cosmos Bank, Sui Coin, etc.)
+3. **Parsing Ambiguity**: Without a standardized format, it's impossible to determine how to parse a token identifier
+
+Universal Token IDs solve these problems by combining a [Universal Chain ID (UCS04)](/ucs/04) with a token kind and denomination into a single canonical identifier.
+
+## Format
+
+```
+<universal_chain_id>:<token_kind>.<denom>
+```
+
+For native tokens that have no denomination:
+
+```
+<universal_chain_id>:<token_kind>
+```
+
+### Components
+
+| Component | Description |
+|-----------|-------------|
+| `universal_chain_id` | A [UCS04 Universal Chain ID](/ucs/04) identifying the chain (e.g., `ethereum.1`, `osmosis.osmosis-1`) |
+| `token_kind` | The token standard used on that chain (see [Token Kinds](#token-kinds)) |
+| `denom` | The token's native denomination or address on its origin chain |
+
+The colon (`:`) separates the chain identifier from the token specification. The period (`.`) separates the token kind from the denomination.
+
+## Token Kinds
+
+| Token Kind | Ecosystem | Has Denom | Description |
+|------------|-----------|-----------|-------------|
+| `erc20` | EVM | Yes | ERC20 tokens on EVM-compatible chains |
+| `evm-native` | EVM | No | Native gas tokens (ETH, MATIC, BNB, etc.) |
+| `cw20` | Cosmos | Yes | CW20 tokens on CosmWasm chains |
+| `cosmos-bank` | Cosmos | Yes | Native Cosmos SDK bank module tokens |
+| `sui-coin` | Sui | Yes | Coin objects on Sui |
+| `sui-token` | Sui | Yes | Token objects on Sui |
+| `starknet-erc20` | Starknet | Yes | ERC20 tokens on Starknet |
+| `starknet-native` | Starknet | No | Native STRK token |
+
+### Native Token Kinds
+
+The following token kinds represent chain-native assets and do **not** include a denomination:
+
+- `evm-native` — The native gas token of an EVM chain (ETH, MATIC, etc.)
+- `starknet-native` — The native STRK token on Starknet
+
+Since each chain has exactly one native token, no disambiguation is required.
+
+## Denomination Rules
+
+### ERC20 Tokens
+
+ERC20 denominations **MUST** be [ERC-55](https://eips.ethereum.org/EIPS/eip-55) checksummed addresses to be considered valid.
+
+```
+ethereum.1:erc20.0x5aAeb6053F3E94C9b9A09f33669435E7Ef1BeAed
+```
+
+An identifier with an incorrectly checksummed address is **invalid**:
+
+```
+ethereum.1:erc20.0x5aaeb6053f3e94c9b9a09f33669435e7ef1beaed  ❌ Invalid (not checksummed)
+ethereum.1:erc20.0x5aAEb6053f3E94c9b9a09f33669435e7ef1beaed  ❌ Invalid (wrong checksum)
+```
```

Comment:
"implementations may choose to parse unchecksummed addresses, as is common when dealing with ethereum addresses, however they must only produce checksummed denoms when displaying."